### PR TITLE
Add new setup.md document

### PR DESCRIPTION
**Pull request by allenkim0129**
# Description

Adding new setup.md document for Python SDK. The setup.md file exist on Java SDK, but it was missing in Python SDK.
- setup.md in JAVA SDK: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/docs/setup.md

This document will explain how to setup Python SDK and run unit tests for new users.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by allenkim0129** [comment]
> @allenkim0129 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Microsoft"

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @allenkim0129

File: sdk/cosmos/azure-cosmos/doc/setup.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+This document currently details Python commands for building Azure Cosmos SDK.
+
+### Fork and clone the repository
+To build and develop locally, it is strongly recommended to fork and clone the repository: https://github.com/Azure/azure-sdk-for-python
```
Comment (bambriz) [inline sdk/cosmos/azure-cosmos/doc/setup.md:4]: I would suggest removing strongly recommended and just directly instructing to fork and clone the repo.

Comment (allenkim0129) [reply sdk/cosmos/azure-cosmos/doc/setup.md]: Done

**Review by simorenoh — APPROVED** [review]
Thank you Allen!

**Pull request merged**